### ISSUE TITLE
Better install manager system. Adding a tab or a queue pop-up for any downloads. 

### ISSUE THREAD

**Issue by shawny43** [modrinth/code#846]
### Is your suggested enhancement related to a problem? Please describe.

Download pop-up show only one content at time and there no queue we can manage.

### Describe the solution you'd like

Adding a download tab similar to Steam where you can manage queue, install and downloads in place. If there some updates, you may see there some modpack that are in pending where you can drag to the queue easily and remove from the queue if you want.

### Describe alternatives you've considered

In another case, you can change the pop-up by an more detailed pop-up like browser does where you can see all your download from a day inside the bubble including queues that's could being shown. 

### Additional context

This download system is crucial for all management. This should being easier to manage and monitor on every clients that implement download and install.

**Comment by srnyx** [comment]
I think even just something as simple as what Thunderstore has would be nice. It has a "Downloads" tab that displays all recent/current mods being downloaded: 
![image](https://github.com/modrinth/theseus/assets/25808801/7a611254-075b-4cb5-b437-d2c683272923)
I've wanted something like this in my Minecraft launcher for a while now cause it makes it 100x easier to figure out which mod is causing a crash/bug after updating all my mods.

**Comment by shawny43** [comment]
Mostly I think is greater but I guess the mod manager have to verify your game if this work before fully complete install to make sure that was not conflict with others mods like Arch Linux does with Aur packages.

**Comment by srnyx** [comment]
> Mostly I think is greater but I guess the mod manager have to verify your game if this work before fully complete install to make sure that was not conflict with others mods like Arch Linux does with Aur packages.

No, sorry, that's not what I meant. I meant a process like this:
1. Update all mods
2. Launch game
3. Game crashes
4. Check the list of mods that updates
5. Perform binary search of **just** those mods instead of every single mod
  a. Or, just check each of these mods' changelogs if it was just 1-5 mods to see if a conflict was mentioned or something

This would prevent having to binary search 100+ mods to just 5-10

**Comment by WhyNineKay** [comment]
I would also like to see a pause and cancel feature for downloads for us with slower internet

**Comment by Prospector** [comment]
it now shows multiple and is toggleable with the download icon at the top
